include/lemlib/devices/encoder/motor.hpp

```diff
@@ -13,16 +14,16 @@ class MotorEncoder : public Encoder {
          * @note change to reference instead of pointer when PROS 4 releases
          *
          * @param motors pointer to the motor group to be used
-         * @param rpm output rpm
+         * @param speed output rpm
```

Comment:
```suggestion
         * @param speed output angular velocity (e.g 200_rpm)
```
Change rpm to angular velocity